Lib9c/Battle/Simulator.cs

```diff
@@ -68,7 +68,7 @@ MaterialItemSheet materialItemSheet
             {
                 try
                 {
-                    var data = itemSelector.Select(1).First();
+                    var data = itemSelector.SelectV2(1).First();
```

Comment:
It is ok for block backwards compatibility before?

Reply:
yes. check this case in `WeightedSelectorTest.SelectV2Single`